src/components/contextMenu/AposContextMenuItem.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <li class="apos-context-menu__item">
+    <button 
+      class="apos-context-menu__button"
+      @click="clicked"
+    >
+      {{ menuItem.label }}
+    </button>
+  </li>
+</template>
+
+
+<script>
+export default {
+  name: 'AposContextMenuItem',
+  props: {
+    menuItem: Object
+  },
+  computed: {
+  },
+  methods: {
+    clicked() {
```

Comment:
This is not a big thing, but since there's a standard `click` event I'd keep that event naming up the chain.